apps/fabric-website/package.json

```diff
@@ -41,13 +41,13 @@
   },
   "dependencies": {
     "@fluentui/examples": "^0.3.23",
+    "@fluentui/theme": "^1.2.0",
```

Comment:
Would you be okay with also updating the markdown files that refer to fluent-theme? (if we're ready to publicly encourage people to use the new theme package)